### DISCUSSION TITLE
scaling down database pods outside office hours

### DISCUSSION THREAD

**Discussion by johanra** [cloudnative-pg/cloudnative-pg#3805] [Q&A]
Hello,
To reduce kubernetes costs I have some kubernetes namespaces that are shutdown outside office hours and in the weekends. To achieve this I am using [kube-downscaler](https://codeberg.org/hjacobs/kube-downscaler) to scale down statefulsets/deployments in selected namespaces. The postgres pods managed by the cloudnative operator are not affected by the kube-downscaler since these are managed directly by the operator and not through a StatefulSet.
Is it possible to integrate the cloudnative operator with kube-downscaler or does the operator provide another way to configure a downtime/uptime window for the pods ?

**Comment by gbartolini** [comment]
Are you looking to keep the persistent volumes and just shut down the pods (that is a sort of hibernation limited to replicas?), or a true scale down approach in which you destroy the persistent volumes too?

**Reply by johanra** [reply]
only the pods need to shutdown that is also what the kube-downscaler does.
